build/Dockerfile.test

```diff
@@ -3,3 +3,4 @@ FROM python:3
 # Install modules
 COPY src/requirements.txt ./
 RUN pip install -r requirements.txt
+RUN python3 -m pip install flake8
```

Comment:
Added `make requirements` to generate requirements.txt based on source.  There's no dependency on flake8 in src so it would be removed.

Reply:
Could append it in the job I guess, would need to know a version.  That could drift away from current over time though.

Reply:
Oh, didn't know requirements.txt was being generated.

Couple of things:
- You don't need a version in a requirements file. You could just have `flake8` on a line by itself and it would work. A lower bound would be nice, though; requirements files accept comparators like `>=`, so e.g. `flake8>=3.7.9` would work fine.
- In OpenStack projects we would maintain a separate requirements file for test (we called it `test-requirements.txt`). We would install both in the test env, but just the non-test one in prod. If you did that here, you could maintain flake8 (and unittest, for that matter -- though I guess you would have to figure out how to make your `requirements.txt` generator ignore test modules) in the `test-requirements.txt`.

Anyway, thanks for the response here, no action required.

Reply:
I proposed the test-requirements.txt thing via PR #48.